js/index.js

```diff
@@ -0,0 +1,26 @@
+const footer = document.createElement('footer');
+const body = document.querySelector('body');
+body.appendChild(footer);
+
+const today = new Date();
+const thisYear = today.getFullYear();
+const copyright = document.createElement('p');
+copyright.innerHTML = `<span>Marilyn \u00A9 ${thisYear}</span>`;
+footer.appendChild(copyright);
+
+const skillsList = ['C++', 'Revit', 'AutoCAD', 'Microsoft Office'];
+const skillsSection = document.getElementById('Skills');
+const skillUL = skillsSection.querySelector('ul');
+
+for (skill of skillsList){
+   let skillItem = document.createElement('li');
+   skillItem.innerHTML = skill;
+   skillItem.style.fontSize = '1.4em';
```

Comment:
Nice job setting styles in your javascript! Another approach would be to define styles in your css and dynamically add the class to your element like you did for the experience items.